plugin.xml

```diff
@@ -75,63 +75,31 @@
       <string>production</string>
     </config-file>
 
-    <!-- Header files -->
```

Comment:
We don't need explicit header declarations on the plugin anymore since all public headers are namespaced and bundled with the dynamic framework library.